### PR TITLE
Feature/node js express

### PR DESCRIPTION
**Pull request by augusticor**
Final exercise of java-script module, usning NodeJS - Express JS, html, css and JS

### CODE REVIEW

File: 15-node-js-express/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+backend/node_modules
+backend/package-lock.json
```
Comment (pushdev-code) [inline 15-node-js-express/.gitignore]:
```suggestion
```
package-lock.json needs to be on the repository for managing the different library versions, it's only a file so it doesn't hurt any storage limit.

File: 15-node-js-express/backend/index.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+'use strict';
+const cafecito_express = require('express');
```
Comment (pushdev-code) [inline 15-node-js-express/backend/index.js]:
```suggestion
const express = require('express');
```
use a better const name. I know it's fun but let's keep it professional all the time.

File: 15-node-js-express/backend/index.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        email: request.body.email,
+        phone: request.body.phone,
+        courses: request.body.courses,
+        message: request.body.message
+    });
+});
+
+server.listen(server.get('port'), () => {
+    console.log(server.get('appName'));
+    console.log(`Now my cafecito server is listening on port ${server.get('port')}`);
```
Comment (pushdev-code) [inline 15-node-js-express/backend/index.js]: check my comment above

**Review by pushdev-code — CHANGES_REQUESTED** [review]
some minor changes, good job.

File: 15-node-js-express/backend/index.js (hunk 2 above)
Comment (augusticor) [reply 15-node-js-express/backend/index.js]: Ok i forgot to change that, jeje was for better learning

**Review by pushdev-code — APPROVED** [review]

**Pull request merged**